### ISSUE TITLE
FeatureConfigurationService.createMissingFeatureConfiguration does not account for duplicate key

### ISSUE THREAD

**Issue by JonasCir** [hzi-braunschweig/SORMAS-Project#4140]
### Bug Description
The problematic code (`FeatureConfigurationService.createMissingFeatureConfiguration:128`) is as following:
``` 
Map<FeatureType, FeatureConfiguration> existingListOfConfigurations =
featureConfigurations.stream().collect(Collectors.toMap(FeatureConfiguration::getFeatureType, Function.identity()));
```
The column `featuretype` in the `featureconfiguration` table can contain duplicate keys (i.e., for LINE_LISTING). This blows up `toMap` when executed with `Duplicate key LINE_LISTING` `Collectors.duplicateKeyException`

The bug was triggered in the update process, so this currently prevents a successful update from `1.52.1` to `1.53`.

### Steps to Reproduce
1. Login as admin on 1.52.1
2. go to settings menu
3. under districts, add two new districts
4. under line listing, activate line listing for all districts (see screenshot)
5.   `select featuretype, district_id from featureconfiguration;
` shows that `LINE_LISTING` is present for each `district_id` (this is not properly handled in the code reading from the DB)
6. do an update to 1.53
7. :boom: 

### Expected Behavior
Upgrade passes

### Screenshots
![line](https://user-images.githubusercontent.com/20031875/105891771-4dbeea00-6011-11eb-8365-60a24f7f7984.png)
![sql](https://user-images.githubusercontent.com/20031875/105891934-7941d480-6011-11eb-8c1e-e0da97856913.png)

### System Details
* Device: SORMAS Docker
* SORMAS version: 1.52.1
* User Role: Admin

### Additional information
Caused by refactoring in #3552

**Comment by StefanKock** [comment]
I consider this to be a blocker for 1.53.0 to be rolled out. But since 1.54.0 is already out it is uncertain if we will fix 1.53 or directly 1.54 (and consider 1.53.0 not to be usable).

**Comment by MateStrysewske** [comment]
The solution to this problem is to ignore feature types that are NOT server features in this code section because those should never be created automatically anyway.

**Comment by Marko-ilmkreis** [comment]
I would agree that for the Health Offices would be rolled-out 1.54 instead 1.53 if the testing does not take again one month. ;-)

**Comment by StefanKock** [comment]
I created a fix based on master:1.54.0

**Comment by AndyBakcsy-she** [comment]
Validated this ticket on the latest version of SORMAS Version: 1.55.0-SNAPSHOT on the local environment.
Did not test step 6. the update
